shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -122,15 +122,6 @@
 }
 
 void FlutterPlatformViewsController::OnCreate(FlutterMethodCall* call, FlutterResult& result) {
-  if (!flutter_view_.get()) {
```

Comment:
Would this race condition be moved to a different part of the code after this change? What about ensuring that `flutter_view_`  isn't `null` in the other parts of the code where it's used?

Reply:
Based on my own auditing of the code, no. We're no longer creating a FlutterView here, we're creating a different UI view that's delegating based on the VC rather than the View.

Reply:
+1.  I think we should make sure we are confident to say that the `flutter_view_` is ready when the platform view is going to be displayed.(when `Rasterizer:Draw` happens)
What about we assert that in `CompositeWithParams`? Or even earlier in `PrerollCompositeEmbeddedView`

Reply:
You can't have a rasterizer without a surface (which the Flutter View creates).

Reply:
Right, I meant that since we are confident that if the Rasterizer:Draw is called, the `flutter_view_` should not nil, we can assert the `flutter_view_` in either `CompositeWithParams` or `PrerollCompositeEmbeddedView`.

Reply:
And the animator won't be started unless you have a surface either - so there won't be any composition/preroll calls without a surface (-> view)